tensorflow/lite/delegates/gpu/cl/BUILD

```diff
@@ -208,9 +208,6 @@ cc_library(
     name = "egl_sync",
     srcs = ["egl_sync.cc"],
     hdrs = ["egl_sync.h"],
-    defines = [
```

Comment:
can you elaborate why this was removed?

Reply:
This was causing linker errors (and that's basically the reason I've switched to loading the `EGL_KHR_fence_sync` functions dynamically) since not all implementations expose the "*KHR" functions. Now that the extension functions are dynamically loaded, is there really a reason to keep this define around?